docs/.vuepress/config.js

```diff
@@ -121,6 +121,11 @@ module.exports = {
                     directory: false,
                     path: "/app_modules/interchain-accounts/integration.html"
                 },
+                {
+                    title: "Authentication module development",
```

Comment:
Nice, I noticed this was missing the other day! I have a [draft](https://github.com/cosmos/ibc-go/pull/719) almost ready for review with some more doc updates/improvements :)